app/code/BelodubrovskyiAn/Lesson11ModuleOop/Block/LessonOopBlock.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace BelodubrovskyiAn\Lesson11ModuleOop\Block;
+
+use BelodubrovskyiAn\Lesson11ModuleOop\Model\FileList;
+use BelodubrovskyiAn\Lesson11ModuleOop\Model\ConstantsMethods;
+use BelodubrovskyiAn\Lesson11ModuleOop\Model\Parameters;
+
+/**
+ * Class LessonOopBlock
+ * @package BelodubrovskyiAn\Lesson11ModuleOop\Block
+ */
+class LessonOopBlock extends \Magento\Framework\View\Element\Template
+{
+    /**
+     * @var FileList
+     */
+    public $fileListGet;
+    /**
+     * @var ConstantsMethods
+     */
+    public $constantsAndMethodsGet;
+
+    public $parametersGet;
```

Comment:
Please add phpdoc for fields and for methods too

Reply:
Fixed.